doc/questions.md

```diff
@@ -0,0 +1,55 @@
+# ros-get questions
+
+- *1. where do these packages come from??*
+  - Would be the ros-distro story, but needs a good place in the manual.
+  - perhaps an example and a eference to the REP.
+
+- *2. which branch?*
+  - ``ros-get update`` does what wrt branches?
+  - should it do other things with branches too? (make them, remove them, switch between them, push them, fetch/pull them, etc)
```

Comment:
Good question, we should discuss this. My personal preference is the following
- install
  - if the repo doesn't exist: checkout the repo @ target branch
  - else: just `git pull` the current branch and warn that you're on the wrong branch
- update
  - if the repo doesn't exist: checkout the repo @ target branch
  - else: just `git pull` the current branch and warn that you're on the wrong branch

Does that make sense?

Reply:
I think it is up to the user to switch branches. And to make sure they're working on the branch they want to work on.
Warnings on an update would be a bit overdone no? If they want to know they can run a ros-get status or something.